### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by J2ZROMERO**
- Add test directory on a logic directory
- Add operate.test.js and calculate.test.js for testing

### CODE REVIEW

File: src/logic/__tests__/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  });
+});
+describe('Testing operate API division', () => {
+  test('The next divition value must be 2', () => {
+    expect(parseInt(operate(10, 5, '÷'), 5)).toBe(2);
+  });
+});
+describe('Testing operate API residue', () => {
+  test('The next residue value must be 2', () => {
+    expect(parseInt(operate(5, 3, '%'), 5)).toBe(2);
```
Comment (OLIPLICHE) [inline src/logic/__tests__/operate.test.js]: - Please make sure all test pass.
- 
Capture1

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @J2ZROMERO, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/logic/__tests__/operate.test.js (hunk 1 above)
Comment (J2ZROMERO) [reply src/logic/__tests__/operate.test.js]: Many thanks for your review @OLIPLICHE

**Comment by J2ZROMERO** [comment]
- fix testing
- Now  all test pass
- Add 10 in the radix parameter

**Review by codecaiine — APPROVED** [review]
Hi @J2ZROMERO @alexmedinasf , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Tests are passed :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**